README.md

```diff
@@ -1,68 +1,28 @@
-# kruize-ui
+# Kruize-UI
 
-This repository contains the code for Kruize Autotune user interface. The objective behind creating the UI is to enable newbies to explore and contribute to Autotune and simplify the initial steps of starting an Autotune experiment. The UI significantly reduces the burden on user to understand, operate and work with Autotune. Not only this it also helps better explain and present what remarkable optimizations Autotune can bring to site reliability engineering.
+This repository contains the code for Kruize user interface. The objective behind creating the UI is to enable users to explore and contribute to Kruize. The UI significantly reduces the burden on user to understand, operate and work with Kruize. Not only this it also helps better represent the remarkable optimizations Kruize can bring to your optimization usecases.
 
-## Quick Start
-
-### Clone Repository
-
-```
-git clone git@github.com:kruize/kruize-ui.git
-```
+This application is built using [Patternfly and React](https://www.patternfly.org/v4/get-started/developers)
 
-### Go to the kruize folder
+## Before You Start
 
-```
-cd kruize-ui
-```
+Kruize UI needs Kruize server running locally in the remote monitoring mode. You can setup Kruize [from here](https://github.com/kruize/kruize-demos/tree/main/remote_monitoring_demo). 
```

Comment:
I think we can get rid of `remote monitoring mode` as it get confusing for users.